README.md

```diff
@@ -1,20 +1,36 @@
 # elasticsearch_docker
-Contains the Elastic Search Dockerfile for Hypothesis' development environment.
 
-## Add a new docker image to the registry
-1. Build the image locally and tag it with the appropriate elastic search version.
-```
-docker build -t hypothesis/elasticsearch:elasticsearch6.2 .
-```
-2. Login to docker hub.
-```
-docker login
-```
-3. Push the docker image to the https://hub.docker.com/r/hypothesis/elasticsearch/ docker registry.
+The Elasticsearch Dockerfile for Hypothesis' development environment.
+
+## Building the Docker image locally
+
+To build the image for just your current platform, use:
+
 ```
-docker push hypothesis/elasticsearch:elasticsearch6.2
+docker build -t hypothesis/elasticsearch:elasticsearch7.10 .
 ```
-4. Cleanup (optional) by deleting any unused images.
+
+To build the image for both x64 and Arm, make sure your Docker environment is
+configured to support [multi-platform
+builds](https://docs.docker.com/build/building/multi-platform/). Then run:
+
 ```
-docker rmi hypothesis/elasticsearch:elasticsearch6.2
+docker buildx build --platform linux/arm64/v8,linux/amd64 --tag hypothesis/elasticsearch:elasticsearch7.10 .
 ```
+
+## Publishing the Docker container
+
+1. Build the container for both x64 and Arm using `docker buildx` as described
+   above.
+
+2. Login to docker hub.
+
+   ```
+   docker login
+   ```
+
+3. Push the Docker image to the registry at https://hub.docker.com/r/hypothesis/elasticsearch/.
+
+   ```
+   docker push hypothesis/elasticsearch:elasticsearch7.10
+   ```
```

Comment:
I have performed these steps already. We could make these steps a task performed by GitHub Actions, although this image is something that gets updated only very rarely.